text/0092-replay-issue-creation.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2023-05-15
+- RFC Type: decision
+- RFC PR: https://github.com/getsentry/rfcs/pull/92
+- RFC Status: draft
+
+# Summary
+
+We want to detect certain categories of issues only available through the Session Replay product. These issues can only be detected on the SDK. The Replay back-end will never have enough data to find these issues. For that reason this is primarily an SDK driven workload. The question is: what role should the Replay back-end have in Replay issue creation? Should the Replay SDK use the Replay back-end to generate new issues or should the SDK generate those issues through a generic, non-replay-specific interface?
+
+# Motivation
+
+1. Inform the long-term strategy for detecting and ingesting Session Replay generated issues.
+2. Provide actionable feedback to developers that could not otherwise be captured in a performance span or error event.
+3. Increase product awareness in free-tier customers and encourage adoption.
+
+# Options Considered
+
+### Option 1: SDK
```

Comment:
Can we have the option names match the wording you used in the Summary section? Or vice versa. It would make it a little easier to follow~

"Should the Replay SDK use the Replay back-end to generate new issues or should the SDK generate those issues through a generic, non-replay-specific interface?"